torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -218,7 +218,9 @@ def __init__(self, info, runtime_metadata, trace_joint):
     def __call__(self, orig_inputs, fw_outs, out):
         aliased_base_tensor = fw_outs[self.base_idx]
         return gen_alias_from_base(
-            aliased_base_tensor,
+            aliased_base_tensor.alias
+            if isinstance(aliased_base_tensor, TensorAlias)
+            else aliased_base_tensor,
```

Comment:
synced offline - one very nice outcome would be if we can **entirely avoid** doing the alias-of-intermediate logic in the repro above. we only actually need to do view replay if we have two graph outputs that are **differentiable** aliases of each other, which is not true in this repro (one is detached)